packages/run-protocol/src/vpool-xyk-amm/constantProduct/invariants.js

```diff
@@ -6,12 +6,12 @@ import { AmountMath } from '@agoric/ertp';
 import { natSafeMath } from '@agoric/zoe/src/contractSupport/index.js';
 
 /**
- * xy <= (x + deltaX)(y - deltaY)
+ * `xy <= (x + deltaX)(y - deltaY)`
```

Comment:
Ok, this was manual. But still curious: Did you do the manual intervention on the input or the output?

Reply:
If the question is whether the formatter would modify it without backticks, yes it would capitalize because `jsdocCapitalizeDescription` is left at its default of `true`.

Alternately I can disable that and we can leave capitalization alone. IMO it's better to enable it to force us to notice what is interpreted as prose, because any rendering of the docstring will render it that way.

Though I can also accept that even if we do agree to that, as a matter of migration we tackle that in a separate stage so that just getting all onto this tool happens faster.